draft-rfcs/RFC-github-access-policy.md

```diff
@@ -0,0 +1,86 @@
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Rust GitHub Permissions Policy
+This policy applies to both the [Rust-Lang GitHub Organization](https://github.com/rust-lang/) and all repositories within that organization.
+
+### Rust-Lang Organization
+Membership in the Rust-Lang GitHub organization is managed by the organization owners.
+
+Only members of the [Core Team](https://github.com/rust-lang/team/blob/master/teams/core.toml) and the [Infrastructure Team](https://github.com/rust-lang/team/blob/master/teams/infra.toml) can be Rust-Lang GitHub Organization owners.
+
+Owners should use a separate account from their main GitHub account dedicated to managing the organization. This account may not be used to commit code and must have 2FA enabled.
+
+### Rust-Lang Repositories
+
+Access to and permissions for repositories within the Rust-Lang organization should be administered through GitHub teams. Rust-Lang GitHub teams are administered through the [Team repository](https://github.com/rust-lang/team).
+
+GitHub provides several permission levels for access to a repository. Please refer to [GitHub's documentation](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization) for details on permission levels and what each level can do.
+
+Repositories in the Rust-Lang organization should follow these permission guidelines:
+
+* **Admin** - only Rust team or working group leads should have this permission level
+* **Write** - contributors within GitHub teams may have this permission level at the discretion of the team leads
+* **Triage** - contributors within GitHub teams involved in issue and pull request management (i.e. closing, re-opening, and assigning issues and pull requests) should be granted this access
+* **Read** - by default, everyone should have access to read repositories
+
+By default, repositories should be public and allow read access to all. When needed, some repositories can have limited read access (i.e. repositories related to security). 
+
+Additionally, the Rust High Five bot should have read access to all repositories in the [Rust-Lang GitHub Organization](https://github.com/rust-lang/).
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
+
+This policy would add more structure to managing GitHub permissions for both the [RustLang](https://github.com/rust-lang). Some might find this structure slows them down and alters their current workflow that they are used to.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+- Why is this design the best in the space of possible designs?
+- What other designs have been considered and what is the rationale for not choosing them?
+- What is the impact of not doing this?
+
+# Prior art
```

Comment:
Is this section necessary for this particular RFC? I suppose it could refer to other open source projects' access policies, but it seems like the benefit might not be worth the level of effort.